lib/repo/subversion_repository.rb

```diff
@@ -565,18 +557,21 @@ def expand_path(file_name, dir_string = "/")
     ##  this class).
     ####################################################################
 
-    # Semi-private class method: Reads in Repository.conf[:REPOSITORY_PERMISSION_FILE]
+    # Semi-private class method
     def self.__read_in_authz_file
       # Check if configuration is in order
-      if Repository.conf[:REPOSITORY_PERMISSION_FILE].nil?
+      if !MarkusConfigurator.markus_config_repository_admin?
+        raise NotAuthorityError.new("Unable to read authsz file:  Not in authoritative mode!");
```

Comment:
Line is too long. [95/80]<br>Do not use semicolons to terminate expressions.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.